src/components/Operator.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const operators = [
+  { name: 'operator plus', value: '+' },
+  { name: 'operator minus', value: '-' },
+  { name: 'operator times', value: '*' },
+  { name: 'operator divide', value: '/' },
+  { name: 'operator equal', value: '=' },
+];
```

Comment:
- [OPTIONAL] It would be better to use the same signs from the Design Guidelines instead of:

![division-1](https://user-images.githubusercontent.com/79876959/144498785-4a49774b-e876-4455-a729-cdfe93d079fd.PNG)

Use this:

![division-2](https://user-images.githubusercontent.com/79876959/144498827-02aef478-954c-4c7c-a8cd-3f2bf5154acc.PNG)